omnibus/Gemfile.lock

```diff
@@ -8,8 +8,8 @@ GIT
 
```

Comment:
We need a `bundle update --conservative` here to get the new version of license_scout

Reply:
Please revert the gemfile before doing the `bundle update` so we get omnibus back to the correct branch as well.